### PR TITLE
Handle Vsap thermal paper in central scan properly

### PR DESCRIPTION
**Pull request by carolinemodic**
## Overview
Update the central scan scanning to:

- Expect a pageHeight of max(bmdThermalPageLength, hmpbPageLength) instead of just hmpbPageLength
- Fill in the image with black background instead of white, this allows hmpb to be cropped as they are today 

Updates the bmd intepretation path to then look everywhere on the page for the qr code. I still break into two "top" and "bottom" search areas as the logic to orient the image relies on that data point. 

Adds a quick benchmark test for bmd ballots. 

Perf numbers from turning on debug logging and looking at the total interpretAndSaveImages times: 

- 22inch ballot election with bmd ballot
  - before: 650ms 
  - after: 880ms
  - change: 230ms
- letter size ballot election with bmd ballot
  - before: 510ms
  - after: 660ms
  - change: 150ms

### CODE REVIEW

File: apps/central-scan/backend/src/fujitsu_scanner.test.ts [hunk 1]
```diff
@@ -93,14 +93,14 @@ test('fujitsu scanner returns ballot audit id on scans when imprinting', async (
   await expect(sheets.scanSheet()).resolves.toBeUndefined();
 });
 
-test('fujitsu scanner can scan with letter size', () => {
+test('fujitsu scanner can scans with expected params on letter size election', () => {
```
Comment (eventualbuddha) [inline apps/central-scan/backend/src/fujitsu_scanner.test.ts:96]: Picking between "can scan" and "scans".
```suggestion
test('fujitsu scanner scans with expected params on letter-size election', () => {
```

File: apps/central-scan/backend/src/fujitsu_scanner.test.ts [hunk 2]
```diff
@@ -116,19 +116,20 @@ test('fujitsu scanner can scan with letter size', () => {
...
       '--page-height',
-      '279.364',
+      '336.506',
+      '--bgcolor=black',
     ])
   );
 });
 
-test('fujitsu scanner can scan with legal size', () => {
+test('fujitsu scanner can scan with expected params on legal size election', () => {
```
Comment (eventualbuddha) [inline apps/central-scan/backend/src/fujitsu_scanner.test.ts:125]: For consistency with letter above.
```suggestion
test('fujitsu scanner scans with expected params on legal-size election', () => {
```

File: apps/central-scan/backend/src/fujitsu_scanner.ts [hunk 3]
```diff
@@ -131,6 +135,7 @@ export class FujitsuScanner implements BatchScanner {
       `--batch=${join(directory, `${dateStamp()}-ballot-%04d.${this.format}`)}`,
       `--batch-print`,
       `--batch-prompt`,
+      `--bgcolor=black`,
```
Comment (eventualbuddha) [inline apps/central-scan/backend/src/fujitsu_scanner.ts:138]: This is a good opportunity to start adding some comments to these arguments. Why is this flag important?

File: libs/ballot-interpreter/src/bmd/utils/qrcode.ts [hunk 4]
```diff
@@ -30,119 +27,24 @@ function decodeBase64FromUtf8(utf8StringData: Buffer): Buffer {
...
-  const bmdHeight = Math.round((size.height * 2) / 5);
-  // We look at the bottom first because we're assuming people will mostly scan
-  // sheets so they appear right-side up to them, but bottom-side first to the
-  // scanner.
-
-  // ┌───┐
-  // │   │
-  // ├─┐ │
-  // └─┴─┘
+  // TODO (#4980) Be more selective about bmd QR code search areas after merging with the hmpb and ensuring images from central scan are properly cropped.
```
Comment (eventualbuddha) [inline libs/ballot-interpreter/src/bmd/utils/qrcode.ts:30]: 👍

**Review by eventualbuddha — COMMENTED** [review]
Don't have time to fully review this morning. Submitting with partial comments.

File: libs/ballot-interpreter/benchmarks/interpret_bmd.bench.ts
Comment (jonahkagan) [inline libs/ballot-interpreter/benchmarks/interpret_bmd.bench.ts:1]: Cool! Glad to have a benchmark for this

**Review by jonahkagan — APPROVED** [review]
lgtm!

**Pull request merged**